### DISCUSSION TITLE
Subdomains stuck in generating SSL certificates

### DISCUSSION THREAD

**Discussion by ajainvivek** [vercel/community#2119] [Help]
### Summary

Hi Team,

Most of our subdomains are stuck in generating SSL certificates. This is blocking most of our customers now. It was working prior to this.

Thanks,
Ajain

### Additional information

<img width="994" alt="Screen Shot 2023-04-19 at 1 03 47 pm" src="https://user-images.githubusercontent.com/1182600/232958671-6aeb1f44-5793-463e-90b9-76738f4ebcd1.png">

**Comment by timothyjoh** [comment]
I was also experiencing this today, just transferring my corporate website over from AWS to Vercel. The WWW domain would not generate an SSL certificate in 5 minutes, so I moved the site back to AWS. How stupid is this? That it will not create SSL **BEFORE** we migrate the CNAME and A record.

This needs to be fixed pronto

**Comment by amyegan** [comment]
@ajainvivek @timothyjoh Are you still having this problem with SSL certificates?
